extensions/amp-subscriptions/amp-subscriptions.md

```diff
@@ -47,160 +74,366 @@ Because of standardization of markup, support for multiple providers, and improv
 support it is recommended that new publisher and paywall provider implementations
 use `amp-subscriptions`.
 
-## Services
+### AMP Reader ID
 
-There could be one or more services configured for `amp-subscriptions`. There could be a local service or vendor services. A local service is configured fully within the page, including its authorization/pingback endpoints, as well as login and subscribe actions. A vendor service is registered as an AMP extension that cooperates with the main `amp-subscriptions` extension.
+To assist access services and use cases, AMP Access introduced the concept of _Reader ID_.
 
-## Product configuration
+The Reader ID is an anonymous and unique ID created by the AMP ecosystem. It is unique for each Reader/Publisher pair - a Reader is identified differently to two different Publishers. It is a non-reversible ID. The Reader ID is included in all AMP/Publisher communications and has very high entropy. Publishers can use the Reader ID to identify the Reader and map it to their own identity systems.
```

Comment:
This sentence is confusing to me:
`a Reader is identified differently to two different Publishers.`

The same reader is given a different id? Or a reader with one id is identified separately by publishers?

Reply:
I do not understand this sentence at all:
`The Reader ID is included in all AMP/Publisher communications and has very high entropy.`

What do you mean by high entropy?

Reply:
>a Reader is identified differently to two different Publishers.

This means that the same reader is represented by different ids to different publishers.

>The Reader ID is included in all AMP/Publisher communications and has very high entropy.

My understanding here is that the reader ID is very obscure to the publisher so cannot be used to try and determine any facts about the user or where they are browsing from. @jpettitt - is that right?